projects/demo/src/modules/customization/dropdown-viewport/examples/import/providers.md

```diff
@@ -0,0 +1,27 @@
+```ts
+import {Component} from '@angular/core';
+import {TUI_VIEWPORT} from '@taiga-ui/core';
+
+@Component({
+  // ...
+  providers: [
+    {
+      provide: TUI_VIEWPORT,
+      useFactory: () => ({
+        type: `viewport`,
+        getClientRect() {
+          return {
+            top: 0,
+            left: 0,
+            right: window.innerWidth,
```

Comment:
Let's add `WINDOW` as a dependency so that it does not reach for global object.

Reply:
fixed